test/fixtures/clone-regexp/case-2/after.js

```diff
@@ -0,0 +1,17 @@
+const regex = /[a-z]/gi;
+
+// Argument is a regex
+const clone1 = new RegExp(/[a-z]/gi)/* Todo */;
+
+// Argument is an identifier
+const clone2 = new RegExp(regex)/* Todo */;
+
+// Both arguments are identifier
+const opt = { ignoreCase: 1 === 1 };
+const clone3 = new RegExp(regex)/* Todo */;
+
+// Multiple cloneRegexp
+const clone4 = f(new RegExp(regex)/* Todo */, new RegExp(regex)/* Todo */);
```

Comment:
in other codemods we add some console.logging for cases that cant entirely be codemodded, if you search for `"[WARNING]"` in the repo you should find something, can we add that here too for consistency?

Reply:
Ah, good to know. I added a warning 👍. They will show up in the test runs though.